src/transaction/types.ts

```diff
@@ -82,6 +82,11 @@ export type Outcome = {
     }[]
   },
   orderbookChanges: object,
+  deliveredAmount?: {
+    currency: string;
+    value: string;
+    counterparty?: string;
+  };
```

Comment:
I think these should be commas rather than semicolons

Reply:
Sorry on missing the comma conventions here. I didn't know that Typescript supports both commas and semicolons as delimiters for `type` and `interface` declarations.

We've been using `semicolons` personally and it appears this has been the case too from their official documentation https://www.typescriptlang.org/docs/handbook/advanced-types.html

Definitely interesting to see different conventions being used.

Reply:
Interesting!